ruby-lsp.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency("language_server-protocol", "~> 3.17.0")
-  s.add_dependency("sorbet-runtime")
+  s.add_dependency("sorbet-runtime", "~> 0.5.11026")
```

Comment:
Agreed, we should, at worst, have a minimum version as @KaanOzkan states above, but one that is not super recent so that folks are inconvenienced by it.